src/Microsoft.Data.Analysis/Microsoft.Data.Analysis.csproj

```diff
@@ -1,15 +1,15 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;net8.0</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;net6.0;net8.0</TargetFrameworks>
```

Comment:
Since `net6.0` is EOL as of November, which is when our next release will be out, we don't need to add it back in as a target framework.

Reply:
To distribute the date32 capability to as many targets as possible, I've added support for the .NET 6 framework. Since date32 requires the DateOnly struct, targeting .NET 6 achieves the widest distribution. Otherwise, any executable that uses this library but targets a version earlier than .NET 8 won't receive the date32 capability, as it would default to the .NET Standard version of the library (which of course won't have the needed struct).

I would appreciate it if we could continue targeting .NET 6, especially since the app I'm using this in also targets .NET 6. If you'd still prefer not to target .NET 6, would it be acceptable to target .NET 7 instead?

Reply:
Fair enough, removed.